onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -8359,6 +8429,44 @@ def _aten_var_mean_dim_onnx(
     return var, mean
 
 
+@torch_op("aten::var", private=True)
+def _aten_var_onnx(self: TReal, correction: float, keepdim: bool = False) -> TReal:
+    mean = op.ReduceMean(self, keepdims=keepdim)
+    sub_mean = op.Sub(self, mean)
+    sqr_mean = op.Mul(sub_mean, sub_mean)
+    var = op.ReduceMean(sqr_mean, keepdims=keepdim)
+    # Adjust var according to correction value
+    if correction > 0.0:
```

Comment:
Can this if branch be  extracted out in trace_only level, so we save it for performance?

Reply:
The if branch can go out because if correction is 0.0 technically it will be N/(N-0.0), but it will require redundant computing.

Reply:
> Can this if branch be extracted out in trace_only level, so we save it for performance?

Don’t trace the branch. Always keep logic as much as possible in an ONNX function.